new-features.md

```
@@ -55,6 +21,55 @@ For the best experience while driving, combine these options to maximize stabili
+===================
+
+## (Done v1.4.0) Search updates (added to advanced search)
+### Bitrate/Speed
+Select from pre-defined bitrate or input field for a specific bitrate.
+
+### Search by Popularity/Votes 
+
+## (Done v1.4.0) Advanced Search update 
+Combined: Use multiple fields to pinpoint exact vibes
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)